pkgs/by-name/cr/cryptpad/0002-don-t-ignore-config-s-httpAddress.patch

```diff
@@ -0,0 +1,46 @@
+From 50f5f3d1f6feb50d9a9b2f24b858f5cad16aef42 Mon Sep 17 00:00:00 2001
+From: Dominique Martinet <asmadeus@codewreck.org>
+Date: Sat, 26 Aug 2023 09:31:46 +0900
+Subject: [PATCH 2/2] don't ignore config's httpAddress
+
+- safe/unsafe port would listen to node's default (0.0.0.0) which is probably
+not intended. Note this change can break existing setups as the default value
+changes to 127.0.0.1 with this.
+- websocket listener hardcoded localhost
+---
+ lib/http-worker.js | 4 ++--
+ server.js          | 2 +-
+ 2 files changed, 3 insertions(+), 3 deletions(-)
+
+diff --git a/lib/http-worker.js b/lib/http-worker.js
+index c3ed9754c3fc..147a7d841423 100644
+--- a/lib/http-worker.js
++++ b/lib/http-worker.js
+@@ -599,9 +599,9 @@ app.use(function (err, req, res /*, next*/) {
+ var server = Http.createServer(app);
+ 
+ nThen(function (w) {
+-    server.listen(Env.httpPort, w());
++    server.listen(Env.httpPort, Env.httpAddress, w());
```

Comment:
I believe this is fixed in Cryptpad 5.6.0. I think it's the other patch about `httpSafePort` in `env.js` that should be applied as per https://github.com/cryptpad/cryptpad/pull/1212

Reply:
I've derived a new patch from the referenced Cryptpad PR by martinetd: https://github.com/michaelshmitty/cryptpad-flake/blob/main/0001-correctly-listen-to-httpSafePort.patch
Should be good, best double check though.